### PR TITLE
Updating for style

### PR DESCRIPTION
**Pull request by esmerel**
Signed-off-by: Lynette Miles <lynette.miles@chronosphere.io>

### CODE REVIEW

File: installation/aws-container.md [hunk 1]
```diff
@@ -1,45 +1,57 @@
 # Containers on AWS
 
-AWS maintains a distribution of Fluent Bit combining the latest official release with a set of Go Plugins for sending logs to AWS services. AWS and Fluent Bit are working together to rewrite their plugins for inclusion in the official Fluent Bit distribution.
+AWS maintains a distribution of Fluent Bit combining the latest official release with
```
Comment (cnorris-cs) [inline installation/aws-container.md:3]:
```suggestion
AWS maintains a distribution of Fluent Bit that combines the latest official release with
```

File: installation/aws-container.md [hunk 2]
```diff
@@ -1,45 +1,57 @@
...
 ## Plugins
 
-Currently, the [AWS for Fluent Bit](https://github.com/aws/aws-for-fluent-bit) image contains Go Plugins for:
+The [AWS for Fluent Bit](https://github.com/aws/aws-for-fluent-bit) image contains Go
+Plugins for:
 
-* [Amazon CloudWatch Logs](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit)
-* [Amazon Kinesis Firehose](https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit)
-* [Amazon Kinesis Streams](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit)
+- Amazon CloudWatch as  `cloudwatch_logs`. See the
```
Comment (cnorris-cs) [inline installation/aws-container.md:13]:
```suggestion
- Amazon CloudWatch as `cloudwatch_logs`. See the
```

File: installation/aws-container.md [hunk 3]
```diff
@@ -1,45 +1,57 @@
...
+  [Fluent docs](https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch) or the
```
Comment (cnorris-cs) [inline installation/aws-container.md:14]: Should these be "Fluent Bit docs" and not "Fluent docs"? Also, these are absolute links. Is that how the links are being handled in the docs?

File: installation/aws-container.md [hunk 4]
```diff
@@ -1,45 +1,57 @@
...
-AWS vends their container image via [Docker Hub](https://hub.docker.com/r/amazon/aws-for-fluent-bit), and a set of highly available regional Amazon ECR repositories. For more information, see the [AWS for Fluent Bit GitHub repo](https://github.com/aws/aws-for-fluent-bit#public-images).
+AWS vends their container image using
+[Docker Hub](https://hub.docker.com/r/amazon/aws-for-fluent-bit), and a set of highly
+available regional Amazon ECR repositories. For more information, see the
+[AWS for Fluent Bit GitHub repository](https://github.com/aws/aws-for-fluent-bit#public-images).
 
-The AWS for Fluent Bit image uses a custom versioning scheme because it contains multiple projects. To see what each release contains, check out the [release notes on GitHub](https://github.com/aws/aws-for-fluent-bit/releases).
+The AWS for Fluent Bit image uses a custom versioning scheme because it contains
+multiple projects. To see what each release contains, see the [release notes on
+GitHub](https://github.com/aws/aws-for-fluent-bit/releases).
```
Comment (cnorris-cs) [inline installation/aws-container.md:38]:
```suggestion
multiple projects. To see what each release contains, see the
[release notes](https://github.com/aws/aws-for-fluent-bit/releases) on GitHub.
```

**Review by cnorris-cs — APPROVED** [review]
I left comments and suggestions, but I've added my approval based on those to keep this moving.

File: installation/aws-container.md (hunk 3 above)
Comment (esmerel) [reply installation/aws-container.md:14]: they mostly seem to be, and I haven't wanted to spend the time struggling with the (working) link structure when I could be fixing things that are actually broken or wrong.

Fighting with the links is a future problem.

Comment (esmerel) [reply installation/aws-container.md]: and yes, I fixed them to say "Fluent Bit docs"

**Pull request merged**